pages/get-started/examples/payments.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Payments
+description: "Unlock the next era of real-time on-chain payments"
+---
+
+#### 🔴 Problems with Payments
+
+- **High Fees & Delays** – Cross-border payments are expensive and slow.
```

Comment:
There seems to be a global issue with bullet points not rendering correctly:

<img width="758" alt="Screenshot 2025-03-25 at 10 20 46" src="https://github.com/user-attachments/assets/9a607673-4169-448e-bd0d-f76fcd1d381b" />

Reply:
Could be on my side, we can double-check in preview